gradle.properties

```diff
@@ -18,4 +18,6 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+# API Key
+API_KEY="027b7152f20f23f8f0cbc9f4bfab880c"
```

Comment:
this should not be here. put this key in the build.gradle

Reply:
Solved